lib/burnside/convert-expr.cc

```diff
@@ -143,15 +146,19 @@ class ExprLowering {
     return createBinaryOp<OpTy>(ex, genval(ex.left()), genval(ex.right()));
   }
 
-  M::FuncOp getFunction(RuntimeEntryCode callee, M::FunctionType funTy) {
-    auto name{getRuntimeEntryName(callee)};
+  M::FuncOp getFunction(const std::string &name, M::FunctionType funTy) {
```

Comment:
Probably want to get used to using llvm::StringRef to pass around string-like data.

Reply:
I could, is StringRef still the golden citizen now that C++17 has `string_view` though ?
https://en.cppreference.com/w/cpp/string/basic_string_view
A nice point of `std::string_view` is that they are constexpr freindly.
I am fine with using llvm::StringRef though.

Reply:
Changed to StringRef in https://github.com/schweitzpgi/f18/pull/3